xbmc/windowing/wayland/WinSystemWaylandWebOS.cpp

```diff
@@ -144,6 +149,39 @@ bool CWinSystemWaylandWebOS::OnAppLifecycleEventWrapper(LSHandle* sh, LSMessage*
   return static_cast<CWinSystemWaylandWebOS*>(context->userdata)->OnAppLifecycleEvent(sh, reply);
 }
 
+void CWinSystemWaylandWebOS::OnConfigure(std::uint32_t serial,
+                                         CSizeInt size,
+                                         IShellSurface::StateBitset state)
+{
+  // intercept minimized event, passing the minimized event causes a weird animation
+  if (state.none())
+  {
+    m_resumePlayback = false;
+
+    const auto& components = CServiceBroker::GetAppComponents();
```

Comment:
maybe move components and player before the if/else condition since it is mandatory in both conditions?

Reply:
perfectly valid point, thanks!